apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -169,6 +173,14 @@ const MeetingForm: React.FC = () => {
         onChange={toggleWebAudio}
         infoText="Enable Web Audio to use Voice Focus"
       />
+      <FormField
+        field={Checkbox}
+        label="Enable Echo Reduction"
+        value=""
+        checked={isEchoReductionEnabled}
+        onChange={toggleEchoReduction}
+        infoText="Enable Echo Reduction (new meetings only)"
+      />
```

Comment:
Use single quote to keep code style consistent.

Reply:
which line should be single quote? I see `Join w/o Audio and Video` and `Enable Web Audio` are all double quotes

Reply:
We should address them as well. : )